AdyenCard/Components/Card/CardPublicKeyFetcherProtocol.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+internal protocol CardPublicKeyFetcherProtocol: PaymentComponent {
```

Comment:
How about `CardPublicKeyRecipient` or something?

Reply:
`CardPublicKeyConsumer` I think is better

Reply:
renamed it to `CardPublicKeyConsumer `

Reply:
👍

Reply:
I think 'Every consumer have a provider' make sense here